crates/spawner/src/draft.rs

```diff
@@ -126,3 +142,69 @@ fn update_draft(
         }
     }
 }
+
+/// Materials for the invalid and valid placing states
+#[derive(Clone, Resource)]
+struct DraftMaterials {
+    valid_placement: Handle<StandardMaterial>,
+    invalid_placement: Handle<StandardMaterial>,
+}
+
+fn insert_materials(mut commands: Commands, mut materials: ResMut<Assets<StandardMaterial>>) {
+    commands.insert_resource(DraftMaterials {
+        valid_placement: materials.add(Color::rgba(0.2, 0.8, 0.2, 0.7).into()),
+        invalid_placement: materials.add(Color::rgba(0.86, 0.08, 0.24, 0.7).into()),
+    });
+}
+
+// Assign the appropriate allowed to all entities in the spawned glb scene
+fn update_object_material(
+    allowed: bool,
+    entities: impl Iterator<Item = Entity>,
+    standard_materials: &mut Query<&mut Handle<StandardMaterial>>,
+    draft_materials: &Res<DraftMaterials>,
```

Comment:
`&Res<DraftMaterials>` -> `&DraftMaterials` would be nicer (you can get it with `res.as_ref()`.